src/Mod/Sketcher/Gui/CommandSketcherBSpline.cpp

```diff
@@ -972,6 +973,157 @@ bool CmdSketcherCompModifyKnotMultiplicity::isActive(void)
     return isSketcherBSplineActive(getActiveGuiDocument(), false);
 }
 
+class DrawSketchHandlerBSplineInsertKnot: public DrawSketchHandler
+{
+public:
+    DrawSketchHandlerBSplineInsertKnot(Sketcher::SketchObject* _Obj, int _GeoId)
+        : Obj(_Obj)
+        , GeoId(_GeoId)
+        , EditMarkers(1)
+    {
+        auto bsp = static_cast<const Part::GeomBSplineCurve *>(Obj->getGeometry(GeoId));
+        guessParam = bsp->getFirstParameter();
+    }
+
+    ~DrawSketchHandlerBSplineInsertKnot() override {}
+
+    void activated(ViewProviderSketch *) override
+    {
+        // TODO: Make a separate cursor for insert knot
+        setCrosshairCursor("Sketcher_Pointer_Create_BSpline");
+    }
+
+    void deactivated(ViewProviderSketch *) override
```

Comment:
Is this empty override function necessary?

Reply:
This one might not be. Copied it straight from one of the `CommandCreateGeo` handlers.

Reply:
Deleted it since it wasn't doing anything and there seem to be some major changes in commands that I wasn't following. Will force commit after addressing some other changes.